source/project-stories/contiki-holidays.erb

```diff
@@ -0,0 +1,399 @@
+    </div>
+  </section>
+
+  <blockquote class="quote">
+    <p class="quote__text">
+      <%= current_page.data.project_quote.quote %>
+    </p>
+    <p class="quote__source">
+      <%= current_page.data.project_quote.source %>
+    </p>
+  </blockquote>
+
+  <section class="component">
+    <div class="component__tile text-tile">
+      <h2 class="text-tile__title">
+        <%= current_page.data.agile.title %>
+      </h2>
+
+      <p class="text-tile__paragraph">
+        <%= current_page.data.agile.content %>
+      </p>
+    </div>
+
+    <div class="component__tile image-tile">
+      <img class="image-tile__image" <%= retina_src current_page.data.agile.image %> alt="<%= current_page.data.agile.image_description %>" />
+    </div>
+  </section>
+
+  <section class="bitesize--dark-gray">
+    <div class="bitesize__tile--two">
+      <p class="bitesize__text--large"><i class="fa fa-arrow-up large-arrow"></i><%= current_page.data.statistics_2.number %><p>
+      <p class="bitesize__text--small"><%=current_page.data.statistics_2.text %></p>
+      <p class="bitesize__text--large"><i class="fa fa-arrow-up large-arrow"></i><%= current_page.data.statistics_3.number %><p>
+      <p class="bitesize__text--small"><%=current_page.data.statistics_3.text %></p>
+    </div>
+  </section>
+
+  <section class="component">
+    <div class="component__tile text-tile">
+      <h2 class="text-tile__title">
+        <%= current_page.data.collaboration.title %>
+      </h2>
+
+      <p class="text-tile__paragraph">
+        <%= current_page.data.collaboration.content %>
+      </p>
+    </div>
+
+    <div class="component__tile image-tile">
+      <img class="image-tile__image" <%= retina_src current_page.data.collaboration.image %> alt="<%= current_page.data.collaboration.image_description %>" />
+    </div>
+  </section>
+
+  <blockquote class="quote">
+    <p class="quote__text">
+      <%= current_page.data.release_quote.quote %>
+    </p>
+    <p class="quote__source">
+      <%= current_page.data.release_quote.source %>
+    </p>
+  </blockquote>
+
+  <section class="component">
+    <div class="component__tile text-tile">
+      <h2 class="text-tile__title">
+        <%= current_page.data.objectives.title %>
+      </h2>
+
+      <p class="text-tile__paragraph">
+        <%= current_page.data.objectives.content %>
+      </p>
+    </div>
+
+    <div class="component__tile image-tile">
+      <img class="image-tile__image" <%= retina_src current_page.data.objectives.image %> alt="<%= current_page.data.objectives.image_description %>" />
+    </div>
+  </section>
+
+  <blockquote class="quote">
+    <p class="quote__text">
+      <%= current_page.data.team_quote.quote %>
+    </p>
+    <p class="quote__source">
+      <%= current_page.data.team_quote.source %>
+    </p>
+  </blockquote>
+
+  <div class="project-team">
```

Comment:
This should probably be a `section`, I realise this is also the same case for SH:24 that I overlooked